modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {navigateToACSitesPageViaURL} from './utils/navigation';
+import {CardSelectors} from './utils/selectors';
+import {changeTimeFilter} from './utils/time-filter';
+import {waitForLoading} from './utils/loading';
+
+import {liferayConfig} from '../../liferay.config';
+import {createIndividuals} from './utils/individuals';
+import {viewNameOnTableList} from './utils/utils';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test('Verify that clicking the "All Search Terms" link redirects the user to the Search Terms tab with the same time filter previously set.', async ({
+	apiHelpers,
+	page,
+}) => {
+	const channelName = 'My Property - ' + getRandomString();
+
+	const {channel, project} = await syncAnalyticsCloud({
+		apiHelpers,
+		channelName,
+		page,
+	});
+
+	const individualName = 'ac';
+
+	const date = new Date();
+
+	const generateIndividual = (name) => {
+		const id = getRandomString();
+
+		return {
+			id,
+			name,
+		};
+	};
+
+	const individuals = [generateIndividual(individualName)];
+
+	await test.step('Create Individuals', async () => {
+		await createIndividuals({
+			apiHelpers,
+			individuals,
+		});
+	});
+
+	await test.step('Create Individuals Events', async () => {
+		const individualsEvent = individuals.map((individual) => ({
+			applicationId: 'Page',
+			canonicalUrl: `https://${liferayConfig.environment.baseUrl}/search`,
+			channelId: channel.id,
+			eventDate: date.toISOString(),
+			eventId: 'pageViewed',
+			title: 'Liferay',
+			url: `https://${liferayConfig.environment.baseUrl}/search?q=test`,
+			userId: individual.id,
+		}));
+
+		await apiHelpers.jsonWebServicesOSBAsah.createEvents(individualsEvent);
```

Comment:
Hey @fmarcio, I think you don't need to create this variable (`individualsEvent`), since it is not being reused in another step. What do you think?